themes/codefor-theme/assets/scss/projects.scss

```diff
@@ -0,0 +1,44 @@
+.project-preview-card {
+  .card-header {
+    height: 30px; // set height explicitly so that cards without icons have the same height as others
+  }
+
+  .project-image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    object-fit: cover;
+    aspect-ratio: 16/9;
+    font-style: italic;
+  }
+}
+
+#projects-container {
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(Min(100%, 400px), 1fr));
+}
+
+.project-accordion-container {
+  display: grid;
+  gap: 10px;
+}
+
+.project-accordion-item {
+  border: none;
+  background: #01a9fa;
```

Comment:
```suggestion
  background: #80ddd7;
```

Reply:
Der Farbkontrast ist mit unserem Standard Blue sehr gering. Das is nicht gut wegen accessibility.  Hier ein Vorschlag mit der Akzentfarbe Light Teal aus dem Styleguide!  (Styleguide als PDF in geteilter Ablage auf der Cloud)

Reply:
Als Alternative kann ich mir auch vorstellen, das hier das Design von landing-callout (der grünen Box) gut passen würde.

Reply:
Habe jetzt mal das light-teal für die header und mint für den Content genommen. Ich glaube das ist ganz hübsch und sollte auch more accessible sein, kannst ja mal gucken, ob es dir gefällt.